bandit_parser.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+#
+# Copyright (C) Citrix Systems Inc.
+#
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Hi @liulinC,

this license conflicts with /LICENSE.
Can this please either be
* changed to 2-clause-BSD
* or include the full GNU Lesser General Public License text in this repo AND specify in /LICENSE. that this file is under a different license than everything else.

Reply:
thanks @robertbreker 
@TalonsLee, could you please help to fix it?